test/index.js

```diff
@@ -132,6 +132,96 @@ describe('bin/rejoice', () => {
         });
     });
 
+    it('composes server with preConnections callback', (done) => {
+
+        const manifest = 'module.exports = { \
```

Comment:
It could be minified or reduced to one long line. Or something like this:

```javascript
const manifest = 'module.exports = {' +
    'connections: [' +
    '    {' +
    '        port: 0,' +
    '        labels: ["api", "nasty", "test"]' +
    '    },' +
    '    {' +
    '        host: "localhost",' +
    '        port: 0,' +
    '        labels: ["api", "nice"]' +
    '    }' +
    '],' +
    'registrations: [' +
    '    {' +
    '        plugin: "./--loaded"' +
    '    }' +
    '],' +
    'preRegister: function(server, next) {' +
    '  console.log("Inside `preRegister` function.");' +
    '  next();' +
    '}' +
'};';
```

Which is compatible with hapi style guide?